.gitignore

```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+dist
```

Comment:
why is this necessary now?

Reply:
After testing I'm not seeing the need for it. Removing but let me know if I missed something.